discovery-provider/compose/.env

```diff
@@ -9,3 +9,4 @@ audius_ipfs_port=5001
 WAIT_HOSTS=discovery-provider-db:5432,redis-server:6379, ipfs-node:5001
 audius_delegate_owner_wallet=0x1D9c77BcfBfa66D37390BF2335f0140979a6122B
 audius_delegate_private_key=0x3873ed01bfb13621f9301487cc61326580614a5b99f3c33cf39c6f9da3a19cad
+audius_discprov_identity_service_url=http://audius-identity-service_identity-service_1:7000
```

Comment:
@piazzatron should the config in creator node get updated to this as well?

Reply:
Yup, nice catch